e2e/suites/entity.test.ts

```diff
@@ -1,10 +1,12 @@
 /*global page*/
 
-import { adminLogin, logout } from './helpers/login';
-import proxyMock from './helpers/proxyMock';
+import { adminLogin, logout } from '../helpers/login';
+import proxyMock from '../helpers/proxyMock';
+import insertFixtures from '../helpers/insertFixtures';
 
 describe('Entities', () => {
   beforeAll(async () => {
+    await insertFixtures();
```

Comment:
Hey @txau, why do we have to run the fixtures on every test suite? Maybe we should include the e2e fixtures script while calling the e2e script? I think this will save a lot of time while running the tests.

Reply:
Actually @daneryl suggested this approach so we know the suites always start at a clean state and can be run independently. Also because of that, we have spent some time optimizing the fixture insertion (ie, by using transpiled scripts) and we removed plenty of document content so indexation is very fast. I think the current times are acceptable?

Reply:
we already had this scenario on the nightmare e2e fixtures where some testSuites fail because they depend on some specific data that the previous test had changed.

current times are good and with transpiled version for running on ci are a lot better, this is definitely an improvement over the old e2e where this same step can take almost 30secs now its more around the 5 secs with the production version.

Reply:
Ok. sounds good